utils/s2n_random.c

```diff
@@ -325,24 +325,43 @@ int s2n_get_rdrand_data(struct s2n_blob *out)
     struct s2n_stuffer stuffer = {{0}};
     union {
         uint64_t u64;
+        struct {
```

Comment:
do you want to wrap this in an

``` if defined(__i386__) ... #endif```

directive?

Reply:
done